andela/lotus-ah-frontend
# #160451571 users should be able to view notifications

#### What does this PR do?

#### Description of Task to be completed?
- write action tests for notifications
- get user unread notifications when authenticated
- mark a single notification as read
- mark all notifications as read

#### How should this be manually tested?
- start the app
- you should see all your notification after you are authenticated
- click on each notification item to clear or
- click on `mark all as read` to clear all

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#160451571](https://www.pivotaltracker.com/story/show/160451571, "view user notification")

#### Screenshots (if appropriate)
<img width="1680" alt="screen shot 2018-11-16 at 3 16 04 pm" src="https://user-images.githubusercontent.com/24890605/48628911-9d78e400-e9b8-11e8-927c-9fb4188f15e8.png">

#### Questions:
